libfwupdplugin/meson.build

```diff
@@ -145,15 +144,24 @@ fwupdplugin_src = [
   'fu-string.c', # fuzzing
   'fu-sum.c', # fuzzing
   'fu-udev-device.c',
-  'fu-usb-device.c',
-  'fu-usb-device-ds20.c', # fuzzing
-  'fu-usb-device-fw-ds20.c', # fuzzing
-  'fu-usb-device-ms-ds20.c', # fuzzing
   'fu-uswid-firmware.c', # fuzzing
   'fu-version-common.c', # fuzzing
   'fu-volume.c', # fuzzing
 ]
 
+if libusb.found()
+  fwupdplugin_src += [
+    'fu-hid-device.c',
```

Comment:
this seems wrong to be gated behind USB support.  hid support could be used for bluez couldn't it?

Reply:
Ahh yes, it's a misnomer. FuHidDevice subclasses FuUsbDevice so this is correct. FuHidDevice should really be an *interface* for both FuUdevDevice and FuUsbDevice I think -- but that's much further down the list of things I want to fix.